src/GitHub.VisualStudio/Services/UIProvider.cs

```diff
@@ -105,7 +134,11 @@ public object TryGetService(Type serviceType)
             if (instance != null)
                 return instance;
 
-            instance = serviceProvider.GetService(serviceType);
+            instance = serviceProvider?.GetService(serviceType);
+            if (instance != null)
+                return instance;
+
+            instance = asyncServiceProvider?.GetServiceAsync(serviceType);
             if (instance != null)
                 return instance;
 
```

Comment:
I want to be clear about the ordering of these loaders, and I need to add some documentation for it, because the ordering is important (and we may use it to override the default MEF exports altogether), so I'm going to clarity and spacing here.